mappings/net/minecraft/client/InteractionManager.mapping

```diff
@@ -1,17 +1,60 @@
 CLASS net/minecraft/class_504 net/minecraft/client/InteractionManager
 	FIELD field_2104 minecraft Lnet/minecraft/client/Minecraft;
+	FIELD field_2105 noTick Z
 	METHOD <init> (Lnet/minecraft/client/Minecraft;)V
 		ARG 1 minecraft
+	METHOD method_1705 resetBlockBreaking ()V
+	METHOD method_1706 update (F)V
+	METHOD method_1707 interactBlock (IIII)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 direction
 	METHOD method_1708 clickSlot (IIIZLnet/minecraft/class_54;)Lnet/minecraft/class_31;
 		ARG 1 windowId
 		ARG 2 slotId
 		ARG 3 button
 		ARG 4 shift
 		ARG 5 player
+	METHOD method_1709 onContainerRemove (ILnet/minecraft/class_54;)V
+		ARG 1 containerId
+		ARG 2 player
+	METHOD method_1710 setWorld (Lnet/minecraft/class_18;)V
+		ARG 1 world
+	METHOD method_1711 preparePlayer (Lnet/minecraft/class_54;)V
+		ARG 1 player
+	METHOD method_1712 interactItem (Lnet/minecraft/class_54;Lnet/minecraft/class_18;Lnet/minecraft/class_31;)Z
+		ARG 1 player
+		ARG 2 world
+		ARG 3 item
+	METHOD method_1713 interactBlockWithItem (Lnet/minecraft/class_54;Lnet/minecraft/class_18;Lnet/minecraft/class_31;IIII)Z
```

Comment:
[interactBlock](https://maven.fabricmc.net/docs/yarn-1.20-pre4+build.2/net/minecraft/server/network/ServerPlayerInteractionManager.html#interactBlock(net.minecraft.server.network.ServerPlayerEntity,net.minecraft.world.World,net.minecraft.item.ItemStack,net.minecraft.util.Hand,net.minecraft.util.hit.BlockHitResult)) in Yarn